core_backend/app/routers/question_answer.py

```diff
@@ -52,18 +52,20 @@ async def llm_response(
     prompt = (
         f"Answer the user query in square brackets:"
         f"\n[{user_query.query_text}]\n"
-        f"in natural language by rewording the following FAQ found below. "
-        f"Address the question directly and do not respond with anything that "
-        f"is outwith the context of the given FAQ."
+        f"by using information from the following FAQ found below:"
+        f"\n\nFound FAQ:\n{content_response[0].response_text}\n\n"
+        f"Address the question directly and briefly (1-2 sentences)"
         f"\nIf the FAQ doesn't seem to answer the question, respond with "
         f"'Sorry, no relevant information found.'"
-        f"\n\nFound FAQ:\n{content_response[0].response_text}"
     )
 
     # generate llm response
     messages = [{"content": prompt, "role": "user"}]
     llm_response_raw = completion(
-        model=OPENAI_LLM_TYPE, messages=messages, temperature=0
+        model=LLM_MODEL,
+        messages=messages,
+        temperature=0,
+        api_base=LLM_ENDPOINT,
```

Comment:
Does litellm call OpenAI API by default? It would take me some time to actually check if that's the case so asking first!